app/views/health_cards/show.html.erb

```diff
@@ -0,0 +1,83 @@
+<%= javascript_pack_tag 'details_pg' %>
+<p id="notice"><%= notice %></p>
+<section class="hero is-link">
+  <div class="hero-body">
+    <p class="title">Patient Details</p>
+  </div>
+</section>
+
+  <section class="section">
+    <div class="container">
+
+      <div class="card is-fullwidth">
+        <header class="card-header">
+          <p class="card-header-title">FHIR Bundle</p>
+          <a class="card-header-icon card-toggle">
+            <i class="fa fa-angle-down"></i>
+          </a>
+        </header>
+          <div class="card-content">
+          <div class="content">
+            <pre style="white-space:pre-wrap;">
+              <p> <%= JSON.pretty_generate(@health_card.to_hash) %> </p>
+            </pre>
+            </div>
+          </div>
+        </div>
+    <br>
+    <br>
+
```

Comment:
I think it would be helpful to add one more dropdown here for the JWS Payload before it is minified (this would correspond with [example b](https://smarthealth.cards/examples/example-00-b-jws-payload-expanded.json) from the spec). Technically you already have the data for that right now displaying under the "FHIR Bundle" toggle.